Core.Collectors/Web/RequestResult.cs

```diff
@@ -6,19 +6,20 @@ namespace Microsoft.CloudMine.Core.Collectors.Web
 {
     public class RequestResult
     {
+        public HttpRequestMessage request;
         public HttpResponseMessage response;
         public IAllowListStatus allowListStatus;
```

Comment:
Any field that is not accessed outside, please make private or private read only.